### ISSUE TITLE
threshold to call homozygous variants

### ISSUE THREAD

**Issue by yusmiatiliau** [jts/nanopolish#423]
Hi Jared,

In the nanopolish variant, what is the threshold to call homozygous variants, and can we change it?

Thanks,
Cen

**Comment by jts** [comment]
Hi,

A variant will be called if nanopolish's model scores it better than the reference sequence. So nanopolish will output variants with low QUAL scores that you might want to filter out. Also, you might want to look at the `SupportFraction` VCF tag as it is very useful for filtering (true variants will have `SupportFraction` close to 1).

Jared

**Comment by yusmiatiliau** [comment]
Hi,

So, there is no definite threshold?
I've got some samples misclassified as heterozygote variants while they are actually homozygotes. And the supportFraction seems to be not so much different from those classified correctly. 

Two examples below are both homozygote variants, but the first one was called as heterozygote. Is there anything I can do with it?

Thanks,


#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
CYP2C1917	176	.	C	T	571.9	PASS	BaseCalledReadsWithVariant=82;BaseCalledFraction=0.66129;TotalReads=111;AlleleCount=1;SupportFraction=0.782111	GT	0/1

CYP2C1917	176	.	C	T	163.8	PASS	BaseCalledReadsWithVariant=23;BaseCalledFraction=0.657143;TotalReads=33;AlleleCount=2;SupportFraction=0.76508	GT	1/1

**Comment by jts** [comment]
No, there is no threshold right now. The SupportFraction for the het is quite high though so I can see why you'd want it called as a hom. I'll think a bit about this.

**Comment by yusmiatiliau** [comment]
Thanks!
